ui/pages/permissions-connect/snap-install/index.scss

```diff
@@ -14,6 +14,10 @@
 
   .permissions-connect-permission-list {
     padding: 0 24px;
+
+    .permission {
+      padding: 8px 0;
```

Comment:
Rather than overriding the padding here, can we change it in `.permissions-connect-permission-list .permission` ? I don't think we want the current padding on any version of the permission list.